src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -352,5 +359,102 @@ void updateUserInfo_shouldUpdateUserDataWithNewChildrenAndPhones() throws Except
                 .andDo(print())
                 .andExpect(jsonPath("$.children", hasSize(1)))
                 .andExpect(jsonPath("$.phones", hasSize(1)));
+            }
+
+    @Test
+    @WithMockUser(username = "trump")
+    @Transactional
+    @DataSet({"database_init.yml", "user/changing_password_or_email/changing_password_or_email_init.yml"})
+    @ExpectedDataSet(value = {"database_init.yml",
+            "user/changing_password_or_email/changing_password_or_email_expected.yml"},
+            ignoreCols = {"password", "email", "lastOnlineTime", "updated"})
```

Comment:
password is encoded. It cannot be compared because every time we get different data. It can be compared only with bCryptPasswordEncoder.matches(String encodedPassword, String password)